Source/Bot/Operations/RedeemInviteOperation.swift

```diff
@@ -69,7 +69,7 @@ class RedeemInviteOperation: AsynchronousOperation {
             return
         }
         Log.debug("Redeeming invite to star \(self.star.feed)...")
-        let queue = OperationQueue.current?.underlyingQueue ?? DispatchQueue.global(qos: .background)
+        let queue = OperationQueue.current?.underlyingQueue ?? DispatchQueue.global(qos: .userInitiated)
```

Comment:
What you are doing here is good, I just wonder if it is the right approach.
In our case, underlyingQueue is always nil because we do not set it in the operation queue. This is something I missed when creating these operations. However, operation queues AND operations have a quality of service property that we can use. 
 The other operations don't mind because they are always created in the same context, but RedeemInvite is used in two different contexts, when sending a mission, or when the user is redeeming an invite through settings. We probably want to use the same quality of service in both contexts but, if we wanted to use different quality of services we couldn't.
So, what I am proposing is using the quality of service operation of the operation when creating the queue here and setting it in the operation queue.

Reply:
Also, all of our operations inherit from AsynchronousOperation, which we do own. So we could create something there that all operations can use so we don't repeate this code everywhere.

Reply:
I updated these operations to use the QoS of the operation 👍 this should be ready for review again.